api/tests/unit/infrastructure/mailers/SendinblueProvider_test.js

```diff
@@ -13,58 +15,130 @@ describe('Unit | Class | SendinblueProvider', () => {
 
   describe('#sendEmail', () => {
 
-    const recipient = 'test@example.net';
+    const senderEmailAddress = 'no-reply@example.net';
+    const userEmailAddress = 'user@example.net';
+    const templateId = 129291;
+
+    let stubbedSibSMTPApi;
+    let mailingProvider;
 
     context('when mail sending is enabled', () => {
 
       beforeEach(() => {
         sinon.stub(mailing, 'enabled').value(true);
         sinon.stub(mailing, 'provider').value('sendinblue');
+
+        sinon.stub(SendinblueProvider, 'createSendinblueSMTPApi');
+        sinon.stub(mailCheck, 'checkMail').withArgs(userEmailAddress).resolves();
+
+        stubbedSibSMTPApi = { sendTransacEmail: sinon.stub() };
```

Comment:
```suggestion
        stubbedSendingblueSMTPApi = { sendTransacEmail: sinon.stub() };
```
J'ai mis du temps à comprendre que Sib = SendingBlue

Reply:
Done